v3/lints/cabf_br/lint_dsa_correct_order_in_subgroup_test.go

```diff
@@ -41,7 +41,7 @@ func TestDSANotCorrectOrderSubgroup(t *testing.T) {
 	pMinusOne.Sub(dsaKey.P, big.NewInt(1))
 	dsaKey.Y = pMinusOne
 	expected := lint.Error
-	out := test.TestLintCert("e_dsa_correct_order_in_subgroup", c)
+	out := test.TestLintCert("e_dsa_correct_order_in_subgroup", c, lint.NewEmptyConfig())
```

Comment:
Is there a reason to pass in an EmptyConfig here every time instead of the Default Config?

Reply:
Hmmm it's because there is actually merge logic at play here. As in, we take a configuration from the frontend, we take the defaults given by the lint, and merge them together with preference give to the frontend. So if an empty confguration is given by the frontend then the lint's default values are used.